README.rst

```diff
@@ -1,5 +1,8 @@
-.. image:: https://github.com/PyCQA/bandit/blob/main/logo/logotype-sm.png
-    :alt: Bandit
+.. raw:: html
+
+    <p align="center">
+        <img src="logo/logotype-sm.png" alt="Bandit">
```

Comment:
Can we use the GitHub URL so this continues to work on PyPI?

Reply:
Sure I can change it back, but I PyPI is currently not rendering it anyway. Maybe due to the URL?